erpnext/accounts/doctype/share_transfer/share_transfer.js

```diff
@@ -16,6 +16,22 @@ frappe.ui.form.on('Share Transfer', {
 				};
 			};
 		});
+		frm.set_query("equity_account", function() {
+			return {
+				filters: {
+					"root_type":"Equity",
```

Comment:
Many times, we don't have any accounts with root_type "Equity" (For example, in India people mostly add equity related accounts under Liability). Hence, it will be better if we allow both Equity and Liability accounts here.

Reply:
@nabinhait 's proposal could be helpful for the future when one wishes to convert bonds issued to stock, or vice versa. I agree, both types should be allowed. Let the user choose, just add a configuration to allow to set the default account type for the filter.